pkg/discovery/worker/group_discovery_worker.go

```diff
@@ -239,7 +224,14 @@ func (worker *k8sEntityGroupDiscoveryWorker) BuildTurboPolicyDTOsFromPolicyBindi
 		case SLOHorizontalScale:
 			groupDTO, err := worker.buildSLOHorizontalScalePolicy(policyBinding)
 			if err != nil {
-				glog.Errorf("Failed to build %v policy: %v.", policyBinding.GetPolicyType(), err)
+				glog.Warningf("Failed to build SLO horizontal scale policy: %v", err)
+				continue
+			}
+			groupDTOs = append(groupDTOs, groupDTO)
+		case ContainerVerticalScale:
+			groupDTO, err := worker.buildContainerVerticalScalePolicy(policyBinding)
+			if err != nil {
```

Comment:
Error is lost here as well.  Maybe this is something for the future.

Feel free to drop if this is likely to be picked up with status work.